interop/src-protobufv3/bin/client.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Can interop and benchmark be also merged?

Reply:
Done, ty